src/app/dataset-view/additional-components/overview-component/overview.component.html

```diff
@@ -49,13 +49,13 @@ <h2 class="box-title align-items-center pt-3">Keywords</h2>
                     </p>
                 </div>
             </ng-template>
-            <div class="my-4" *ngIf="canAddSetPollingSource" data-test-id="add-set-polling-source-block">
+            <div class="my-4" data-test-id="add-set-polling-source-block">
```

Comment:
Why deleted condition?

Reply:
It was for testing, forgot to revert, and already moved it back